test/test_linalg.py

```diff
@@ -988,6 +988,26 @@ def test_eigh_errors_and_warnings(self, device, dtype):
             with self.assertRaisesRegex(RuntimeError, "tensors to be on the same device"):
                 torch.linalg.eigh(a, out=(out_w, out_v))
 
+    @skipCPUIfNoLapack
+    @dtypes(torch.float, torch.double)
+    @precisionOverride({torch.float32: 1, torch.float64: 1})
```

Comment:
Might make more sense to put some sane relative errors, as we are comparing rather large numbers.

Reply:
I thought precisionOverride is atol only? Let me check the code.